frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -732,7 +739,7 @@ export default function SurveyEdit(): JSX.Element {
                                                                     groups: [
                                                                         {
                                                                             properties: [],
-                                                                            rollout_percentage: undefined,
+                                                                            rollout_percentage: 50,
```

Comment:
added 50 as the default value for the rollout percentage

Reply:
why not default to 100?  I feel like generally people treat these targeting conditions as booleans (on/off).

Reply:
updated